prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullName = 'Marc Ting';
+    console.log(fullName);
+    console.log(typeof fullName);
+    var numberOfPets = 1;
```

Comment:
make sure to declare all of the variables using `const` except for the never variable, which should be declared with `let`.

Reply:
I made changes to the variables to the variables. Thank you for the feedback!
<img width="346" alt="Screen Shot 2023-02-28 at 5 19 03 PM" src="https://user-images.githubusercontent.com/93560875/222020241-b3f3de8d-f7be-4d2a-bf0e-9140b8e78951.png">

Reply:
Attached is also a picture of the terminal output
<img width="815" alt="Screen Shot 2023-02-28 at 6 17 32 PM" src="https://user-images.githubusercontent.com/93560875/222028148-99b1db3a-1373-4f51-bf38-8016672aa17a.png">